README.md

```diff
@@ -1,30 +1,30 @@
 # Godot UI theme
 
-UI themes for godot applications.
+UI themes for godot engine.
 
-## For editor
+## For the editor
 
-Themes can be applied to Godot editor after install.
+The themes can be applied to the Godot editor after installation.
 
 * Run `python install.py` to install the themes.
-* Select the theme path end with `.tres` under your install directory in godot editor with `Settings/Global/Custom Theme`.
+* Select the theme path ending with `.tres` under your install directory in the godot editor with `Settings/Global/Custom Theme`.
 
 You can pass a param to run `install.py` to set install directory.
 
 ### Default install directory
 
-It should put into the `theme` folder under your `.godot` folder.
+It should install it into the `theme` folder under your `.godot` folder.
 
```

Comment:
I don't exactly know what you mean? Do you think it is bad that the headline is mentioning: "install" and the the sentence too?